pkg/storegateway/bucket_test.go

```diff
@@ -2267,10 +2259,12 @@ func testBucketStoreSeriesBlockWithMultipleChunks(
 					if streamingBatchSize == 0 {
 						require.Zero(t, estimatedChunks)
 					} else {
-						require.Equal(t, testData.expectedChunksEstimate, estimatedChunks)
+						promChunks := queryPromSeriesChunkMetas(t, mimirpb.FromLabelAdaptersToLabels(seriesSet[0].Labels), promBlock)
+						promChunks = filterPromChunksByTime(promChunks, testData.reqMinTime, testData.reqMaxTime)
+						require.InDelta(t, len(promChunks), estimatedChunks, 0.1, "number of chunks estimations should be within 10% of the actual number of chunks on disk")
```

Comment:
Could we compare `estimatedChunks` to the number of chunks in `seriesSet[0].Chunks`? Or is there a reason why we need to read the chunks from disk?

Reply:
good point. I don't think there's a reason to read from disk

Reply:
did that in 1277b1590